src/components/package-show/package-show.js

```diff
@@ -4,8 +4,26 @@ import { Link } from 'react-router-dom'
 import Paneset from '@folio/stripes-components/lib/Paneset';
 import Pane from '@folio/stripes-components/lib/Pane';
 import KeyValue from '@folio/stripes-components/lib/KeyValue';
+import List from '@folio/stripes-components/lib/List';
+import styles from './package-show.css';
+
+export default function PackageShow({ vendorPackage, packageTitles }) {
+  const renderTitleListItem = item => (
+    <li key={item.titleId} data-test-eholdings-package-details-title>
+      <h5 data-test-eholdings-package-details-title-name>
+        {item.titleName}
+      </h5>
+      <div data-test-eholdings-package-details-title-selected>
+        { /* assumes that customerResourcesList.length will always equal one */  }
+        {item.customerResourcesList[0].isSelected ? (
+          <span>Selected</span>
+        ) : (
+          <span>Unselected</span>
+        )}
+      </div>
+    </li>
+  );
```

Comment:
we should probably start splitting these presentational components up a little bit soonish, but that's probably its own discussion